### PR TITLE
Android - fix MyPaths mistake about http decoding

### PR DESCRIPTION
**Pull request by kursatbakis**

### CODE REVIEW

File: source/android/Renaissance/lib/models/path.dart [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import 'package:portakal/models/basic_path.dart';
 import 'package:portakal/models/milestone.dart';
 
-class Path {
+class Pathh {
```
Comment (Hazerank) [inline source/android/Renaissance/lib/models/path.dart:4]: It looks like there is a typo in the name of the path class. You should fix it first Kürşat.

**Review by Hazerank — CHANGES_REQUESTED** [review]
This http decoding was a real pain in the neck. Thanks for your perfect work while fixing it. There is only one typo at the name of the class. After then we are okay to go!

File: source/android/Renaissance/lib/models/path.dart (hunk 1 above)
Comment (fihlie) [reply source/android/Renaissance/lib/models/path.dart:4]: I actually kinda like it.

**Review by fihlie — APPROVED** [review]
I don't have any feedback. However, I've also changed the token and course container so we should be careful when merging these two branches

**Pull request merged**